### ISSUE TITLE
Installation loop between 6.2.5 and 6.3.0

### ISSUE THREAD

**Issue by th-lamb** [MuhammedKalkan/OpenLens#110]
Version 6.2.5 shows update 6.3.0. After update, the section "Pods" under "Workloads" is gone. When restarting the app (including "Quit App" in the task bar), the old version 6.2.5 is started. Pods are available again and the application shows an update.

The whole process repeats again and again.

OS: Windows 10 Enterprise, 22H2, Build 19045.2486

![2023-01-18_143653](https://user-images.githubusercontent.com/50012757/213188198-7e1fb390-78f8-4e2e-badd-fffb6da72b78.png)
![2023-01-18_144000](https://user-images.githubusercontent.com/50012757/213188244-2263ae90-a54b-4c94-b39e-ea361a27dd89.png)
![2023-01-18_144122](https://user-images.githubusercontent.com/50012757/213188271-cec85126-31e2-4352-b87a-ec1051104cfc.png)
![2023-01-18_144205](https://user-images.githubusercontent.com/50012757/213188297-2cd47af9-c72c-403d-a83f-a0085be8948b.png)

**Comment by robervexcel** [comment]
https://github.com/MuhammedKalkan/OpenLens/issues/81

**Comment by hyprsnpr** [comment]
I'm now trying the latest alpha (4) and I have tried the others as well as 6.3.0. The only versions that seem to work are 6.2.5 and below. Yes I have looked at the issue 81 post and yes I have added the extension "npm i @alebcay/openlens-node-pod-menu", however the menus are still missing.

![image](https://user-images.githubusercontent.com/35845818/213703209-86438a94-1731-4121-991c-e055d68a42cd.png)

![image](https://user-images.githubusercontent.com/35845818/213703310-71113f33-b0ce-47cc-a334-5641d9968329.png)

![image](https://user-images.githubusercontent.com/35845818/213703386-e5dd6aba-c4e6-4771-bf79-9c35abfa66bf.png)
